### PR TITLE
release/1.0.1

### PR DESCRIPTION
**Pull request by Zethson**
1.0.1 (2020-11-03)
------------------

**Added**

**Fixed**

* cookietemple lint workflow does no longer try to run autopep8 for python projects

**Dependencies**

**Deprecated**

### CODE REVIEW

File: .github/workflows/sync_project.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: cookietemple sync
```
Comment (Imipenem) [inline .github/workflows/sync_project.yml:1]: why is this necessary? did we missed the file in initial release?

Comment (Zethson) [reply .github/workflows/sync_project.yml:1]: cookietemple itself did not have the sync workflow :)

File: cookietemple/lint/lint.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 log = logging.getLogger(__name__)
 
 
-def lint_project(project_dir: str, is_create: bool = False) -> TemplateLinter:
+def lint_project(project_dir: str, skip_external: bool, is_create: bool = False) -> TemplateLinter:
```
Comment (Imipenem) [inline cookietemple/lint/lint.py:17]: add a quick param explanation

Comment (Zethson) [reply cookietemple/lint/lint.py:17]: it's not like there's one for is_create :)

File: .github/workflows/sync_project.yml (hunk 1 above)
Comment (Imipenem) [reply .github/workflows/sync_project.yml:1]: Ah makes sense ty

File: cookietemple/lint/lint.py (hunk 2 above)
Comment (Imipenem) [reply cookietemple/lint/lint.py:17]: Yeah haha Just saw it, sry, I should have done this before
